### ISSUE TITLE
[documentation] Banner to indicate documentation version

### ISSUE THREAD

**Issue by maximikos** [jump-dev/HiGHS.jl#191]
Hi,

I have been browsing the documentation quite a bit recently, often by having been directed there following a Google search. I noticed that no info/warning is given when one is in the docs for an older version. For example, after a search for how to get the basis with HiGHS.jl, I was directed to [version 0.3.2](https://docs.juliahub.com/HiGHS/WFic5/0.3.2/) of the docs.

Now, to be frank, I haven't checked what changes exactly between versions (does an explicit change log exist somewhere, by the way?). And obviously one can see the current version of the docs in the bottom left.

In any case, however, I believe that it could be useful to have a notification to the user that a more recent version exists, similarly to how it is done with the banner in the docs for JuMP (e.g. [v1.7](https://jump.dev/JuMP.jl/v1.7/)).

Thanks.

**Comment by odow** [comment]
I deeply share your annoyance. The version 0.3.2 docs you linked are hosted by `docs.juliahub.com`, which is run by JuliaHub (a private company unrelated to JuMP-dev). I do not have control over what they host. I've had a number of conversations with them about this, and I think it is a very negative thing for the ecosystem for them to continue to host out-dated documentation. Let me try again.

The "official" docs are the GitHub readme https://github.com/jump-dev/HiGHS.jl#readme, or the JuMP documentation: https://jump.dev/JuMP.jl/stable/packages/HiGHS/.

I'll take the blame for any confusing documentation under `jump.dev` domain, and I'll happily implement any improvements you can suggest :smile:

**Comment by odow** [comment]
I've talked to the folks at JuliaHub. They're aware of the issue, and are going to take a look at adding the banner (apparently it should be added by default, so something went wrong in this case). I'll keep pestering them until something changes.

But for now, there's not much we can do from the point of HiGHS.jl, unfortunately.

**Comment by maximikos** [comment]
Oh, ok. I didn't know that! Thanks for clarifying and taking charge of that :)
